config/logit/filters.d/99_clean_alb_access.conf

```diff
@@ -0,0 +1,14 @@
+#
+# Fix up the datapoints for the alb_access log type
+#
+if [@input] == "alb_access" {
+    ## remove extraneous fields
+    mutate {
+        remove_field => [ '[@shipper][name]', '[@source][component]', '[@source][type]' ]
```

Comment:
I think we quite regularly filter on `@source.component` in Kibana. It would be nice if we could have a meaningful value in that field for ALB access logs.